odoo/tools/cloc.py

```diff
@@ -60,6 +61,8 @@ def parse_js(self, s):
         # Based on https://stackoverflow.com/questions/241327
         s = s.strip() + "\n"
         total = s.count("\n")
+        if max(len(l) for l in s.split('\n')) > MAX_LINE_SIZE:
```

Comment:
```suggestion
        # To avoid to use too much memory we don't try to count file 
        # with very large line, usually minified file
        if max(len(l) for l in s.split('\n')) > MAX_LINE_SIZE:
```